src/System.Security.Cryptography.X509Certificates/tests/PropsTests.cs

```diff
@@ -139,6 +139,7 @@ public static void TestGetPublicKey()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp1_0, "dotnet/corefx#12235")]
```

Comment:
Note that I removed these attributes as 1.0 tests are not currently running any longer. I need to follow up on this however they were not broken because of anything done in this PR